web/styles/popovers.css

```diff
@@ -1098,20 +1126,36 @@ ul {
     }
 }
 
-.personal-menu-actions {
+.navbar-dropdown-menu {
     display: flex;
     flex-flow: column nowrap;
+    margin: 0;
+    padding: 0;
```

Comment:
The padding value is overriden by `#gear-menu-dropdown`, so that property can be safely dropped, too.

Reply:
Yeah, done.